lib/MusicBrainz/Server/Data/Artist.pm

```diff
@@ -119,6 +119,46 @@ sub find_by_area {
     );
 }
 
+sub find_by_instrument {
+    my ($self, $instrument_id, $limit, $offset) = @_;
+
+    # NOTE: if more tables than l_artist_recording are added here, check admin/BuildSitemaps.pl
```

Comment:
This comment looks wrong/irrelevant here, so should be removed (we select from more than l_artist_recording below, and since this is a new page/has no JSON-LD markup, it wouldn't be in the sitemaps listing anyway).